src/main/java/edu/cornell/kfs/fp/businessobject/PurchasingDataBase.java

```diff
@@ -0,0 +1,525 @@
+    public void setAccountNumber(String accountNumber) {
+      this.accountNumber = accountNumber;
+    }
+
+
+    /**
+     * @return the tsysTranCode
+     */
+    public String getTsysTranCode() {
+      return tsysTranCode;
+    }
+
+    /**
+     * @param tsysTranCode the tsysTranCode to set
+     */
+    public void setTsysTranCode(String tsysTranCode) {
+      this.tsysTranCode = tsysTranCode;
+    }
+
+    /**
+     * @return the itemCommodityCode
+     */
+    public String getItemCommodityCode() {
+      return itemCommodityCode;
+    }
+
+
+    /**
+     * @param itemCommodityCode the itemCommodityCode to set
+     */
+    public void setItemCommodityCode(String itemCommodityCode) {
+      this.itemCommodityCode = itemCommodityCode;
+    }
+
+
+    /**
+     * @return the merchantOrderNumber
+     */
+    public String getMerchantOrderNumber() {
+      return merchantOrderNumber;
+    }
+
+
+    /**
+     * @param merchantOrderNumber the merchantOrderNumber to set
+     */
+    public void setMerchantOrderNumber(String merchantOrderNumber) {
+      this.merchantOrderNumber = merchantOrderNumber;
+    }
+
+
+    /**
+     * @return the discountAmount
+     */
+    public KualiDecimal getDiscountAmount() {
+      return discountAmount;
+    }
+
+    /**
+     * @param discountAmount the discountAmount to set
+     */
+    public void setDiscountAmount(KualiDecimal discountAmount) {
+      this.discountAmount = discountAmount;
+    }
+
+    /**
+     * @param freightShippingAmount the discountAmount to set
+     */
+    public void setDiscountAmount(String discountAmount) {
+      if (StringUtils.isNotBlank(discountAmount)) {
+          this.discountAmount = new KualiDecimal(discountAmount);
+      } else {
+          this.discountAmount = KualiDecimal.ZERO;
+      }
+    }
+
+    /**
+     * @return the freightShippingAmount
+     */
+    public KualiDecimal getFreightShippingAmount() {
+      return freightShippingAmount;
+    }
+
+
+    /**
+     * @param freightShippingAmount the freightShippingAmount to set
+     */
+    public void setFreightShippingAmount(String freightShippingAmount) {
+      if (StringUtils.isNotBlank(freightShippingAmount)) {
+          this.freightShippingAmount = new KualiDecimal(freightShippingAmount);
+      } else {
+          this.freightShippingAmount = KualiDecimal.ZERO;
+      }
+    }
+
+
+    /**
+     * @param freightShippingAmount the freightShippingAmount to set
+     */
+    public void setFreightShippingAmount(KualiDecimal freightShippingAmount) {
+      this.freightShippingAmount = freightShippingAmount;
+    }
+
+
+    /**
+     * @return the dutyAmount
+     */
+    public KualiDecimal getDutyAmount() {
+      return dutyAmount;
+    }
+
+
+    /**
+     * @param dutyAmount the dutyAmount to set
+     */
+    public void setDutyAmount(KualiDecimal dutyAmount) {
+      this.dutyAmount = dutyAmount;
+    }
+
+
+    /**
+     * @param freightShippingAmount the freightShippingAmount to set
```

Comment:
Minor mismatch between Javadoc and method, probably not worth fixing unless other items come up in the review.